README.md

```diff
@@ -1,24 +1,11 @@
-# README
+# GOV.UK Forms - API
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
+`forms-api` is the API for the GOV.UK Forms platform.
 
-Things you may want to cover:
+# Environment variables to set
 
-* Ruby version
-
-* System dependencies
-
-* Configuration
-
-* Database creation
-
-* Database initialization
-
-* How to run the test suite
-
-* Services (job queues, cache servers, search engines, etc.)
-
-* Deployment instructions
-
-* ...
+| Name                                        | Purpose                                                 |
+| ------------------------------------------- | ------------------------------------------------------- |
+| `SETTINGS__FORMS_API__AUTHENTICATION_KEY`   | The API key                                             |
+| `SETTINGS__FORMS_API__SENTRY__DSN`          | The DSN provided by Sentry                              |
+| `SETTINGS__FORMS_API__SENTRY__ENVIRONMENT`  | The Sentry environemnt Sentry                           |
```

Comment:
We shouldn't need these because everything should be documented in the settings yaml file. We will need to mention in general terms that settings exist and where to configure them but we cna add that later.